src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+
+type Props = {
+  addNewMovie: (newMovie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addNewMovie }) => {
+  const [query, setQuery] = useState('');
+  const [hasError, setHasError] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
Better to follow these rules to make your code cleaner and more understandable for other developers:
![image](https://user-images.githubusercontent.com/96409563/211368968-cf35b1d4-cb8c-4295-abbc-36c9420b712b.png)